libtsuba/src/RDGManifest.h

```diff
@@ -87,6 +89,11 @@ class KATANA_EXPORT RDGManifest {
 
   bool IsEmptyRDG() const { return num_hosts() == 0; }
 
+  void ResetVersion() {
+    version_ = 1;
+    previous_version_ = 0;
+  }
```

Comment:
This is only used in copy RDG I would honestly rather just put this logic there than give them impression that this is something other should do.

Reply:
So I was debating that and I would have to create setter functions to set the version and previous_version_ in this manner. Would that be better?

Reply:
I'm not sure what that would look like.

Reply:
Ok, I'm going to add a TODO here for now. We can mention this in the rework of tsuba.